tests/index.js

```diff
@@ -96,7 +97,17 @@ describe('node-insights', function(){
     insights.send();
     setTimeout(function(){
       expect(scope.isDone()).to.be.true;
-      expect(body).to.eql('[{"eventType":"purchase","purchase.account":3,"purchase.amount":259.54}]');
+
+      var parsedBody = JSON.parse(body);
+      expect(parsedBody).to.have.length(1);
+
+      var processedInsight = parsedBody[0];
+      expect(Object.keys(processedInsight).sort()).to.eql(['eventType', 'purchase.account', 'purchase.amount', 'timestamp'].sort());
+      expect(processedInsight.eventType).to.eql('purchase');
+      expect(processedInsight['purchase.account']).to.eql(3);
+      expect(processedInsight['purchase.amount']).to.eql(259.54);
+      expect(processedInsight.timestamp).to.be.at.least(addTime);
+
```

Comment:
This style is more verbose but should be just as precise and allows for variable data to be added (e.g. the `timestamp` field)